.travis.yml

```diff
@@ -0,0 +1,15 @@
+# R for travis: see documentation at https://docs.travis-ci.com/user/languages/r
+
+os: osx
+language: r
+r: 3.6.3
+r_github_packages: jasp-stats/jaspTools
```

Comment:
Could you use https://github.com/jasp-stats/jasp-desktop/blob/stable/JASP-Tests/README.md#travisyml?